CaImAn/caiman_run.py

```diff
@@ -54,17 +54,24 @@
 except SyntaxError:
     utils.print_to_intercept(ADVANCED_BAD_TYPE)
     sys.exit()
+except Exception as error:
+    #utils.print_error(error)
```

Comment:
Should be commented?

Reply:
good question and I don't know..... I want to add a message with it but I don't know what message... :(

Reply:
🤷‍♀️